model/patch_lifecycle.go

```diff
@@ -327,16 +325,14 @@ func GetPatchedProjectConfig(ctx context.Context, settings *evergreen.Settings,
 		return "", errors.Errorf("patch '%s' already finalized", p.Version)
 	}
 
-	if p.ProjectStorageMethod != "" || p.PatchedParserProject != "" {
+	if p.ProjectStorageMethod != "" {
 		// If the patch has been created but not finalized, it has either
-		// already saved the project config as a string (if any), stored the
-		// parser project document (i.e. ProjectStorageMethod), or has the
-		// entire patched parser project as a string (i.e.
-		// PatchedParserProject). Since the project config is optional, the
-		// patch may be created and have already evaluated the patched project
-		// config, but there simply was none. Therefore, this is a valid way to
-		// check and get the PatchedProjectConfig after the patch is already
-		// created.
+		// already saved the project config as a string (if any) or stored
+		// the parser project document (i.e. ProjectStorageMethod). Since
+		// the project config is optional, the patch may be created and have
+		// already evaluated the patched project config, but there simply
+		// was none. Therefore, this is a valid way to check and get the
+		// PatchedProjectConfig after the patch is already created.
 		return p.PatchedProjectConfig, nil
 	}
 
```

Comment:
Small suggestion to mention here in a doc comment that if we get past this point, then the patch has not been created yet, so this is the first-time initialization to get the parser project and project config.

Reply:
done